oddbird/Metecho
# consistent capitalization of core objects/terms

![](https://source.unsplash.com/featured/?cute,animal&!!!)




## Description
- Makes core terms (Epic, Task, Project, Scratch/Dev/Test Org, Project etc) consistent in how they are capitalized. 
- Also fixes some wayward punctuation that falls outside of quotation marks ( e.g. "A wayward period". )


## Steps to test/reproduce
- with the environment running search for the above terms
- Otherwise, look throughout the running project (toast messages, modals, etc.) to see updated capitalization
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._


## Show me
_Provide screenshots/animated gifs/videos if necessary._
